### PR TITLE
Fix for even samples

### PR DESCRIPTION
**Pull request by Matt-Ord**

### CODE REVIEW

File: surface_potential_analysis/kernel/kernel.py [hunk 1]
```diff
@@ -510,37 +555,24 @@ def get_noise_operators_isotropic(
         pad_ft_points(kernel["data"], (n,), (0,)),
         norm="forward",
     )
```
Comment (Winnie08L) [inline surface_potential_analysis/kernel/kernel.py:557]: why is coefficients obtained via inverse fft but not fft of beta?

Comment (Matt-Ord) [reply surface_potential_analysis/kernel/kernel.py:557]: This is just to match the convention used in the original paper

File: surface_potential_analysis/kernel/kernel.py [hunk 2]
```diff
@@ -691,19 +727,18 @@ def get_noise_operators_real_isotropic_stacked(
...
-        ) / np.sqrt(2)
+        end = n // 2
+        cos_slice = slice_along_axis(slice(1, end + 1), axis)
+        conj_cos_slice = slice_along_axis(slice(-1, end - 1, -1), axis)
+
+        sin_slice = slice_along_axis(slice(end + 1, None), axis)
+        conj_sin_slice = slice_along_axis(slice(end - 1, 0, -1), axis)
+
+        data[cos_slice] = (cloned[cos_slice] + cloned[conj_cos_slice]) / np.sqrt(2)
+        data[sin_slice] = (cloned[conj_sin_slice] - cloned[sin_slice]) / np.sqrt(2)
```
Comment (Winnie08L) [inline surface_potential_analysis/kernel/kernel.py:741]: operators in the form of e^(ikx), so sin is cloned[sin_slice]-cloned[conj]?

Comment (Matt-Ord) [reply surface_potential_analysis/kernel/kernel.py:741]: Yeah it doesn't effect the final noise kernel, but I'll change it to make things more obvious

**Comment by Winnie08L** [comment]
can i approve this now?

**Comment by Matt-Ord** [comment]
> can i approve this now?

Yeah as long as you approve of it 😃 - just approve and click squash and merge

**Review by Winnie08L — APPROVED** [review]

**Pull request merged**